### PR TITLE
Add new IcoswISC30E3r1 mesh

### PR DESCRIPTION
**Pull request by xylar**
See https://github.com/MPAS-Dev/compass/pull/667 for details on the mesh and E3SM initial condition

Includes changes #58 and #59 to spare ourselves a merging/rebasing nightmare.

### CODE REVIEW

File: components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml [hunk 1]
```diff
@@ -456,6 +500,9 @@
 <config_btr_dt ocn_grid="WCAtl12to45E2r4">'0000_00:00:15'</config_btr_dt>
 <config_btr_dt ocn_grid="SOwISC12to60E2r4">'0000_00:00:15'</config_btr_dt>
 <config_btr_dt ocn_grid="ECwISC30to60E2r1">'0000_00:01:00'</config_btr_dt>
+<config_btr_dt ocn_grid="ECwISC30to60E3r1">'0000_00:01:00'</config_btr_dt>
+<config_btr_dt ocn_grid="IcoswISC30E3r1">'0000_00:01:00'</config_btr_dt>
```
Comment (xylar) [inline components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml:504]:
```suggestion
<config_btr_dt ocn_grid="IcoswISC30E3r1">'0000_00:00:45'</config_btr_dt>
```
Even with a Haney number of 10, I wasn't able to get a stable `simulation` run with 1 minute `btr_dt` so I think this is going to have to be 45 seconds unless more spin-up allows us to relax this.

**Comment by jonbob** [comment]
@xylar - I'm mostly through with making the mapping and domain files. Do you want me to add this mesh to config_grids.xml? Or do you have a script that does that?

**Comment by xylar** [comment]
I did that manually. If you want to get started, go for it. If you want me to do it, that's fine, too.

Thanks for making the mapping files!

**Comment by jonbob** [comment]
I'll add the parts to config_grids.xml as I add the files -- thanks

**Comment by xylar** [comment]
@jonbob, I'm going to rebase this so Icos30 comes after EC30to60 because RRS isn't quite there yet.

**Comment by xylar** [comment]
Closing in favor of #64

**Pull request closed without merging**